multimodal/tests/unittests/test_save_path.py

```diff
@@ -0,0 +1,69 @@
+import os
+import shutil
+import pytest
+from autogluon.multimodal import MultiModalPredictor
+
+from unittest_datasets import PetFinderDataset
+
+
+@pytest.mark.parametrize(
+    "save_path",
+    [
+        "an_existing_path",
+        "~/an_existing_path",
+        os.path.join(os.path.expanduser("~"), "an_existing_path"),
+    ],
+)
+def test_existing_save_path(save_path):
+    abs_path = os.path.abspath(os.path.expanduser(save_path))
+    os.makedirs(abs_path, exist_ok=True)
+    with pytest.raises(ValueError):
+        predictor = MultiModalPredictor(path=save_path)
+
+    dataset = PetFinderDataset()
+    predictor = MultiModalPredictor(
+        label=dataset.label_columns[0],
+        problem_type=dataset.problem_type,
+        eval_metric=dataset.metric,
+    )
+    with pytest.raises(ValueError):
+        predictor.fit(train_data=dataset.train_df, save_path=save_path)
+
+    shutil.rmtree(abs_path)
+
+
+def test_continuous_training_save_path():
```

Comment:
Can we have multiple functions to test different use cases of continuous training? Like (1) one predictor, but continue to train on new data (2) load predictor and resume training. And also, is it possible to load optimizer states beyond just model weights? Thanks.

Reply:
These two cases are covered in this test (Lines 62 and 66). Continuous training is different from resuming training. Resuming training means the previous `fit()` doesn't finish and we resume from `last.ckpt`. Continuous training means the previous `fit()` finish successfully and a model checkpoint `model.ckpt` is saved. Thus, continuous training only loads the model weights without optimizer states.